src/Row/RowItem.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
This probably should have been broken out into its own file in the first place, but was necessary for this documentation feature because react-docgen associates _all_ proptypes it finds in a file with the default exported component.